Announce opam 2.3.0~beta2 (#108)
ocaml/platform-blog: opam-2-3-0-beta2.md

title: "opam 2.3.0 beta2 release"
authors: [
  "Raja Boujbel - OCamlPro" {"mailto:raja.boujbel(à)ocamlpro.com"}
  "Kate Deplaix - Ahrefs" {"mailto:kit-ty-kate(à)outlook.com"}
  "David Allsopp - Tarides" {"mailto:david(à)tarides.com"}
]
date: "2024-10-23"
--BODY--

_Feedback on this post is welcomed on [Discuss](https://discuss.ocaml.org/t/ann-opam-2-3-0-beta2/XXXXX)!_

We're happy to announce the second beta release of opam 2.3.0.

> Compared to the previous 2.3.0\~beta1 release, you can view the full list of changes in the

Author: tildes are not interpreted by opam2web's markdown parser?

> Compared to the previous 2.3.0\~beta1 release, you can view the full list of changes in the

Reviewer: nop, cf https://opam.ocaml.org/blog/opam-2-3-0-beta1/ (i fixed it)

> Compared to the previous 2.3.0\~beta1 release, you can view the full list of changes in the

Author: Ah i see:
```ocaml
(* omd 1.3 *)
# Omd.of_string "2.3.0\\~beta2";;
- : Omd.t = [Omd.Paragraph [Omd.Text "2.3.0\\~beta2"]]
```
```ocaml
(* omd 2.0 *)
# Omd.of_string "2.3.0\\~beta2";;
- : Omd.doc = [Omd.Paragraph ([], Omd__.Ast_inline.Text ([], "2.3.0~beta2"))]
```

[release note](https://github.com/ocaml/opam/releases/tag/2.3.0-beta2).

This version is a beta, we invite users to test it to spot previously

> unnoticed bugs as we head towards the stable release.

Author: i'm not sure. I'd rather say stable release because this the final goal.
I think that, when you're "heading towards <smth>" there can be many intermediate steps.

> unnoticed bugs as we head towards the stable release.

Reviewer: ok

## Changes

> * Fix a regression in the detection of the current terminal size ([#6243](https://github.com/ocaml/opam/issues/6243))

Author: ```suggestion
* Fix a regression in the detection of the current terminal size that leads to opam output that tries to fit itself into 80 columns regardless of the current terminal size ([#6243](https://github.com/ocaml/opam/issues/6243))
```

> 

Author: ```suggestion

* A NetBSD/x86_64 pre-built release binary is now available

```

> Compared to the previous 2.3.0\~beta1 release, you can view the full list of changes in the
> [release note](https://github.com/ocaml/opam/releases/tag/2.3.0-beta2).
> 
> This version is a beta, we invite users to test it to spot previously
> unnoticed bugs as we head towards the stable release.
> 
> ## Changes
> 
> * Fix a regression in the detection of the current terminal size ([#6243](https://github.com/ocaml/opam/issues/6243))
> 
> API changes are denoted in the release note linked above.

Reviewer: proposal (it doesn't integrate the other proposals)
```suggestion
Compared to the previous 2.3.0~beta1 release, the main change is:
* Fix a regression in the detection of the current terminal size ([#6243](https://github.com/ocaml/opam/issues/6243))

You can view the full list of changes in the
[release note](https://github.com/ocaml/opam/releases/tag/2.3.0-beta2), including API ones.

This version is a beta, we invite users to test it to spot previously
unnoticed bugs as we head towards the stable release.
```

> You can view the full list of changes in the
> [release note](https://github.com/ocaml/opam/releases/tag/2.3.0-beta2),
> including API ones.

Author: ```suggestion
You can view the full list of changes, including API changes, in the
[release note](https://github.com/ocaml/opam/releases/tag/2.3.0-beta2).
```

> Compared to the previous 2.3.0~beta1 release, you can view the full list of changes in the
> [release note](https://github.com/ocaml/opam/releases/tag/2.3.0-beta2).
> 
> This version is a beta, we invite users to test it to spot previously
> unnoticed bugs as we head towards the stable release.
> 
> ## Changes
> 
> * Fix a regression in the detection of the current terminal size that leads to opam output that tries to fit itself into 80 columns regardless of the current terminal size ([#6243](https://github.com/ocaml/opam/issues/6243))
> 
> API changes are denoted in the release note linked above.

Reviewer: ```suggestion
Compared to the previous 2.3.0~beta1 release, the main change is
* Fix a regression in the detection of the current terminal size that leads to opam output that tries to fit itself into 80 columns regardless of the current terminal size ([#6243](https://github.com/ocaml/opam/issues/6243)).

You can view the full list of changes in the
[release note](https://github.com/ocaml/opam/releases/tag/2.3.0-beta2),
including API ones.

This version is a beta, we invite users to test it to spot previously
unnoticed bugs as we head towards the stable release.
```

Author: I'm not sure about removing the `## Changes` section. Maybe we can readd `## Changes` and move "This version is a beta back where it was before the `## Changes` section?

Reviewer: like that ? 

---
This version is a beta, we invite users to test it to spot previously
unnoticed bugs as we head towards the stable release.

## Changes

Compared to the previous 2.3.0~beta1 release, the main change is
* Fix a regression in the detection of the current terminal size that leads to opam output that tries to fit itself into 80 columns regardless of the current terminal size ([#6243](https://github.com/ocaml/opam/issues/6243)).

You can view the full list of changes in the
[release note](https://github.com/ocaml/opam/releases/tag/2.3.0-beta2),
including API ones.

---

Author: looks better to me. Let's apply this change so it's easier to improve it further
